pkg/virt-config/feature-gates.go

```diff
@@ -19,73 +19,15 @@
-	VirtIOFSGate          = "ExperimentalVirtiofsSupport"
-
-	DownwardMetricsFeatureGate = "DownwardMetrics"
-	Root                       = "Root"
-	ClusterProfiler            = "ClusterProfiler"
-	WorkloadEncryptionSEV      = "WorkloadEncryptionSEV"
-	VSOCKGate                  = "VSOCK"
-	// DisableCustomSELinuxPolicy disables the installation of the custom SELinux policy for virt-launcher
-	DisableCustomSELinuxPolicy = "DisableCustomSELinuxPolicy"
-	// KubevirtSeccompProfile indicate that Kubevirt will install its custom profile and
-	// user can tell Kubevirt to use it
-	KubevirtSeccompProfile = "KubevirtSeccompProfile"
-	// DisableMediatedDevicesHandling disables the handling of mediated
-	// devices, its creation and deletion
-	DisableMediatedDevicesHandling = "DisableMDEVConfiguration"
-	// PersistentReservation enables the use of the SCSI persistent reservation with the pr-helper daemon
-	PersistentReservation = "PersistentReservation"
-	// VMPersistentState enables persisting backend state files of VMs, such as the contents of the vTPM
-	VMPersistentState = "VMPersistentState"
-	MultiArchitecture = "MultiArchitecture"
-	// VMLiveUpdateFeaturesGate allows updating certain VM fields, such as CPU sockets to enable hot-plug functionality.
-	VMLiveUpdateFeaturesGate = "VMLiveUpdateFeatures"
-	// NetworkBindingPlugingsGate enables using a plugin to bind the pod and the VM network
-	// Alpha: v1.1.0
-	// Beta:  v1.4.0
-	NetworkBindingPlugingsGate = "NetworkBindingPlugins"
-	// AutoResourceLimitsGate enables automatic setting of vmi limits if there is a ResourceQuota with limits associated with the vmi namespace.
-	AutoResourceLimitsGate = "AutoResourceLimitsGate"
-
-	// AlignCPUsGate allows emulator thread to assign two extra CPUs if needed to complete even parity.
-	AlignCPUsGate = "AlignCPUs"
-
-	// VolumesUpdateStrategy enables to specify the strategy on the volume updates.
-	VolumesUpdateStrategy = "VolumesUpdateStrategy"
-	// VolumeMigration enables to migrate the storage. It depends on the VolumesUpdateStrategy feature.
-	VolumeMigration = "VolumeMigration"
-	// Owner: @xpivarc
-	// Alpha: v1.3.0
-	//
-	// NodeRestriction enables Kubelet's like NodeRestriction but for Kubevirt's virt-handler.
-	// This feature requires following Kubernetes feature gate "ServiceAccountTokenPodNodeInfo". The feature gate is available
-	// in Kubernetes 1.30 as Beta.
-	NodeRestrictionGate = "NodeRestriction"
-	// DynamicPodInterfaceNaming enables a mechanism to dynamically determine the primary pod interface for KuveVirt virtual machines.
-	DynamicPodInterfaceNamingGate = "DynamicPodInterfaceNaming"
-)
-
 func (config *ClusterConfig) isFeatureGateEnabled(featureGate string) bool {
-	deprecatedFeature := deprecation.FeatureGateInfo(featureGate)
-	if deprecatedFeature != nil && deprecatedFeature.State == deprecation.GA {
+	deprecatedFeature := featuregate.FeatureGateInfo(featureGate)
+	if deprecatedFeature != nil && deprecatedFeature.State == featuregate.GA {
```

Comment:
The `deprecatedFeature` var name is misleading now. I'd suggest:

```
func (config *ClusterConfig) isFeatureGateEnabled(featureGateName string) bool {
	featureGate := featuregate.FeatureGateInfo(featureGateName)
	if featureGate != nil && featureGate.State == featuregate.GA {
```

Reply:
Done